content/posts/2025-09-03_qchat_redesign.md

```diff
@@ -0,0 +1,134 @@
+---
+title: The redesign of QChat
```

Comment:
It's much more, no?
Some ideas to reflect that:

- the QChat evolution
- the best QChat ever deployed
- QChat just got better